packages/map/src/helpers/getStatesPicked.ts

```diff
@@ -3,9 +3,12 @@ import { supportedStatesFipsCodes } from '../data/supported-geos'
 
 export const getStatesPicked = (config, runtimeData) => {
   const stateNames = getFilterControllingStatesPicked(config, runtimeData)
-
-  return stateNames.map(stateName => ({
-    fipsCode: Object.keys(supportedStatesFipsCodes).find(key => supportedStatesFipsCodes[key] === stateName),
-    stateName
-  }))
+  return stateNames.map(stateName => {
+    const matchedState = Object.values(supportedStatesFipsCodes).find(name => name === stateName)
+    if (!matchedState) console.error(`State name "${stateName}" not found in supportedStatesFipsCodes.`)
+    return {
+      fipsCode: matchedState,
```

Comment:
@joshlacey should this remain FIPS or is the state name fine here?

Reply:
@adamdoe  I changed the variable name for clarification.